src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -118,28 +119,39 @@ let rec mkKn (ty: Type) =
 
 let mutable logger: ILogger = Abstractions.NullLogger.Instance
 
-let loadProject toolsPath properties projPath =
+let loadProjects toolsPath properties (projPaths: string list) =
     async {
+        let paths =
+            projPaths
+            |> List.map (fun proj -> Path.Combine(Environment.CurrentDirectory, proj) |> Path.GetFullPath)
+
+        for proj in paths do
+            logger.LogInformation("Loading project {0}", proj)
+
         let loader = WorkspaceLoader.Create(toolsPath, properties)
-        let parsed = loader.LoadProjects [ projPath ] |> Seq.toList
+        let projectOptions = loader.LoadProjects projPaths
```

Comment:
This is one of the major speed ups. Load all projects at once.

There's some locking that goes on behind the scenes when ProjInfo talks to msbuild becuase msbuild can only handle one build request at a time in process. Tt's best to front load as many project load requests as you can.